jobs/build/release/Jenkinsfile

```diff
@@ -42,6 +42,12 @@ node {
                         $class: 'hudson.model.StringParameterDefinition',
```

Comment:
Just gonna leave the old `ADVISORY` param in there for now?

Reply:
I'd say yes. We can get this URL from the ET API eventually.